Modules/vc-module-export/VirtoCommerce.ExportModule.Core/Model/ExportedTypeMetadata.cs

```diff
@@ -8,45 +8,76 @@ namespace VirtoCommerce.ExportModule.Core.Model
 {
     public class ExportedTypeMetadata : ValueObject
     {
+        /// <summary>
+        /// Extended type info. Stores if specific property is entity (true) or not (false)
+        /// </summary>
+        private class ExportTypePropertyInfoEx
+        {
+            public ExportTypePropertyInfo PropertyInfo { get; set; }
+            public bool IsEntity { get; set; }
+        }
+
         public string Version { get; set; }
         public ExportTypePropertyInfo[] PropertiesInfo { get; set; }
 
+        /// <summary>
+        /// Returns basic info about potentially exported fields of entity
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <returns></returns>
         public static ExportedTypeMetadata GetFromType<T>()
         {
             var result = new ExportedTypeMetadata();
             var t = typeof(T);
```

Comment:
Please rename `t` to `type`, and `pi` to `propertyInfo`. We do not use such names (exception - i, j in cycles and x in lambda), as using full names increases readability.